spec/bolt/cli_spec.rb

```diff
@@ -73,6 +73,11 @@
       expect(cli.parse).to include(nodes: %w[foo bar])
     end
 
+    it "accepts multiple nodes across multiple declarations" do
+      cli = Bolt::CLI.new(%w[command run --nodes foo,bar --nodes more,bars])
+      expect(cli.parse).to include(nodes: %w[foo bar more bars])
```

Comment:
Can you add the same node name to both `--nodes` flags to verify the uniqueness is working?